common/dynamicconfig/constants.go

```diff
@@ -522,6 +522,8 @@ const (
 	ArchivalProcessorArchiveDelay = "history.archivalProcessorArchiveDelay"
 	// ArchivalProcessorRetryWarningLimit is the number of times an archival task may be retried before we log a warning
 	ArchivalProcessorRetryWarningLimit = "history.archivalProcessorRetryLimitWarning"
+	// ArchivalBackendMaxRPS is the maximum rate of requests per second to the archival backend
```

Comment:
where is the rate limiter used?

Reply:
It's instantiated here: 
<img width="560" alt="image" src="https://user-images.githubusercontent.com/5942963/208996855-38602f28-ff79-4ffc-8148-6c3b1db7d22a.png">

And then it's used here: 
<img width="709" alt="image" src="https://user-images.githubusercontent.com/5942963/208996966-0f9ea1aa-4280-4559-8a04-b4c9c382cf70.png">